modules/nw-sriov-network-attachment.adoc

```diff
@@ -38,45 +54,49 @@ metadata:
   namespace: openshift-sriov-network-operator <2>
 spec:
   networkNamespace: <target_namespace> <3>
-  ipam: <ipam> <4>
+  resourceName: <sriov_resource_name> <4>
   vlan: <vlan> <5>
-  resourceName: <sriov_resource_name> <6>
-  linkState: <link_state> <7>
-  maxTxRate: <max_tx_rate> <8>
-  minTxRate: <min_rx_rate> <9>
-  vlanQoS: <vlan_qos> <10>
-  spoofChk: "<spoof_check>" <11>
+  spoofChk: "<spoof_check>" <6>
+ifdef::ocp-sriov-net[]
+  ipam: <ipam> <7>
+  linkState: <link_state> <8>
+  maxTxRate: <max_tx_rate> <9>
+  minTxRate: <min_rx_rate> <10>
+  vlanQoS: <vlan_qos> <11>
   trust: "<trust_vf>" <12>
   capabilities: <capabilities> <13>
+endif::ocp-sriov-net[]
 ----
 <1> Replace `<name>` with a name for the CR. The SR-IOV Network Operator creates a NetworkAttachmentDefinition CR with same name.
 <2> Specify the namespace where the SR-IOV Operator is installed.
 <3> Optional: Replace `<target_namespace>` with the namespace where the NetworkAttachmentDefinition CR is created. The default value is `openshift-sriov-network-operator`.
-<4> Optional: Replace `<ipam>` a configuration object for the ipam CNI plug-in as a YAML block scalar. The plug-in manages IP address assignment for the attachment definition.
+<4> Replace `<sriov_resource_name>` with the value for the `.spec.resourceName` parameter from the SriovNetworkNodePolicy CR that defines the SR-IOV hardware for this additional network.
```

Comment:
if ipam is not provided, installation failed when creating vm's. I am not sure if same is the case with other optional values as well.

Reply:
@geetikakay @phoracek 
What I understand is that ipam must me present in the config but empty?
ie `ipam: {}` 
Will return ipam to this example as such. Please correct me if that's mistaken.

Reply:
@aburdenthehand  There is a bugzilla raised for it. It is not working the way it is mentioned.
https://bugzilla.redhat.com/show_bug.cgi?id=1858223